src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -2136,6 +2137,12 @@ private void displayWorkflowComments() {
             }
         }
     }
+    
+    private void displayPublishMessage(){
+        if (workingVersion.isDraft() && workingVersion.getId() != null && canUpdateDataset()){
```

Comment:
In the case that workingId is = null, when is that? When you're creating a new version?  Why is that scenario different? (may have an obvious answer)

Reply:
it's on dataset create.  the check permissions fails when the id is null.